NitroxServer/GameLogic/Entities/EntitySimulator.cs

```diff
@@ -93,10 +109,17 @@ private List<Entity> AssignForCells(Player player, AbsoluteEntityCell[] added)
             {
                 List<Entity> entities = entityData.GetEntities(cell);
 
-                assignedEntities.AddRange(
-                    entities.Where(entity => cell.Level <= entity.Level &&
-                                                ((entity.SpawnedByServer && serverSpawnedSimulationWhiteList.Contains(entity.TechType)) || !entity.SpawnedByServer) &&
-                                                simulationOwnershipData.TryToAcquire(entity.Guid, player, DEFAULT_ENTITY_SIMULATION_LOCKTYPE)));                       
+                foreach (Entity entity in entities)
+                {
+                    if (cell.Level <= entity.Level && simulationOwnershipData.TryToAcquire(entity.Guid, player, DEFAULT_ENTITY_SIMULATION_LOCKTYPE))
+                    {
+                        assignedEntities.Add(entity);
+                        assignedEntities.AddRange(
+                        entity.ChildEntitiesByGuid.Values.Where(childEntity => cell.Level <= childEntity.Level &&
+                                                    ((childEntity.SpawnedByServer && serverSpawnedSimulationWhiteList.Contains(childEntity.TechType)) || !childEntity.SpawnedByServer) &&
+                                                    simulationOwnershipData.TryToAcquire(childEntity.Guid, player, DEFAULT_ENTITY_SIMULATION_LOCKTYPE)));
```

Comment:
A more general organizational question pops up here: does it make sense to separate parent and child simulation+ownership?